packages/client/window/src/utils/popupStrategy.ts

```diff
@@ -6,20 +6,8 @@ interface CreatePopupService {
 export class CreatePopupStrategy {
     protected createPopupService: CreatePopupService;
 
-    constructor() {
-        this.createPopupService = this.isCrossmintOrigin()
-            ? new CrossmintPopupService()
-            : new CrossOriginPopupService();
-    }
-
-    private isCrossmintOrigin(): boolean {
-        try {
-            const url = new URL(window.location.origin);
-            return url.hostname.endsWith("crossmint.com");
-        } catch (e) {
-            console.error("Invalid URL", e);
-            return false;
-        }
+    constructor(crossOrigin: boolean) {
```

Comment:
maybe add `crossOrigin: boolean = false` as it previously defaulted to false in the catch block